devtools/startup/DevToolsStartup.jsm

```diff
@@ -1804,13 +1807,14 @@ function reloadAndRecordTab(gBrowser) {
   });
 }
 
+let startRecordingTime;
```

Comment:
The rest of the patch looks good, but can we compute the recording duration in the recording process itself?  Using this global variable in the UI process will behave incorrectly if there are multiple recording tabs.